examples/connext_dds/basic_security/py/README.md

```diff
@@ -0,0 +1,85 @@
+# Example Code: Basic Security
+
+## Running this Example
+
+### Initial Setup
+
+In order to run this example you must have security certificates. The easiest
+way to acquire them is to copy them from the rti_workspace examples.
+They can be found in `<rti_workspace>/<version>/examples/dds_security`.
+If this folder is copied in whole to examples/connext_dds/basic_security the
+example will work with no modifications to the SecureQos.xml file.
+Otherwise, the strings in SecureQos.xml must be updated to reflect the locations
+of the certificates.
+
+### Linux and macOS systems
+
+Your LD_LIBRARY_PATH or (DYLD_LIBRARY_PATH for macOS) must include
+`$NDDSHOME/lib/<architecture>`.
+You must also include the path to your crypto libraries. They are in
+`$NDDSHOME/third_party/openssl-<version>/<architecture>/<release or debug>/lib`
+(location of libcrypto.so and libssl.so).
+If you are using the Security Plugins for use with wolfSSL, your
+LD_LIBRARY_PATH must include
+`$NDDSHOME/third_party/wolfssl-<version>/<architecture>/<release or debug>/lib`
+(location of libwolfssl.so).
+
+If using Certicom Security Builder Engine for QNX, your LD_LIBRARY_PATH must
+include
+`$NDDSHOME/third_party/openssl-<version>/<architecture>/release/lib/:$CERTICOM_SBENGINEHOME/tools/sb/sb-$CERTICOMOS/lib/:$CERTICOM_SBENGINEHOME/lib/$CERTICOMOS.`
+
+To run this example, type the following commands in two different command
+shells (one command in each shell), either on the same machine or on different
+machines:
+
+  > python3 secure_pub.py
+  > python3 secure_sub.py
+
+### Windows systems
+
+Your PATH must include
+`%NDDSHOME%\lib\<architecture>` and
+`%NDDSHOME%\third_party\openssl-<version>\<architecture>\<release or debug>\bin`
+(location of the libcrypto and libssl DLLs).
+
+To run this example, type the following commands in two different command
+shells (one command in each shell), either on the same machine or on different
+machines:
+
+  > python3 secure_pub.py
+  > python3 secure_sub.py
```

Comment:
Nit-picking: Should we remember here to use a version of python that we support? (linking to wherever the requirements are specified). In the python API documentation I can see:

> The RTI Connext Python API provides access to most Connext features from Python and is supported on Python 3.6 to 3.11, and on Linux® x64, Windows® x64, and macOS® x64.

Reply:
I agree, the idea was to reference the place where we document the versions we support.
In any case, not important enough to hold the PR.